packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -903,19 +903,19 @@ export class DeltaManager
 
     public emitDelayInfo(
         id: string,
-        delaySeconds: number,
+        delayMs: number,
         error: ICriticalContainerError,
     ) {
-        const timeNow = Date.now();
+        const timeNow = Date.now() * 1000;
```

Comment:
timenow is already ms: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

Reply:
@vladsud  looks like another retry bug

Reply:
we were adding seconds to ms when computing timeTillThrottling, so retrying over aggressively